node/validation/manifest/index.test.ts

```diff
@@ -97,32 +97,57 @@ describe('bundle', () => {
 })
 
 describe('route', () => {
+  let freshValidateManifest: typeof validateManifest
```

Comment:
We'll be able to get rid of this once we remove the feature flag, right?

Reply:
Yes this is only because the code only initializes ajv once, but with the FF changing during the tests we have to reset the modules.